lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -707,15 +710,27 @@ def _get_wala_distro_version(self, node: Node) -> str:
 
         return result
 
+    def _get_boot_time(self, node: Node) -> Dict[str, Any]:
+        boot_time: Dict[str, Any] = {}
+        try:
+            systemd_analyze_tool = node.tools[SystemdAnalyze]
+            boot_time = systemd_analyze_tool.get_boot_time()
+            boot_time["boot_type"] = node.execute(
```

Comment:
What's the output of boot_type? How about put it in system analyze or kernel tool?

Reply:
I may change the column into boot_times, means first boot or second etc